flutter/lib/src/integrations/app_start/app_start_tracker.dart

```diff
@@ -0,0 +1,41 @@
+import 'dart:async';
+
+import 'package:meta/meta.dart';
+
+import '../../../sentry_flutter.dart';
+
+@internal
+class AppStartInfo {
+  final DateTime start;
+  final DateTime end;
+  final SentryMeasurement measurement;
```

Comment:
hm yeah good point I agree

Reply:
I guess one reason why I added it was because then I didn't have to take care of checking whether it's a cold or warm start. so we would need to add something like `isColdStart` boolean to app start info, wdyt?